google/ads/googleads/client.py

```diff
@@ -36,7 +36,10 @@
 
 # Retrieve the version of this client library to be sent in the user-agent
 # information of API calls.
-_CLIENT_INFO = ClientInfo(client_library_version=metadata.version("google-ads"))
+try:
+    _CLIENT_INFO = ClientInfo(client_library_version=metadata.version("google-ads"))
+except metadata.PackageNotFoundError:
+    _CLIENT_INFO = ClientInfo()
```

Comment:
Just noting that if this exception is hit, the side effect is that the client library version will be null in the request payload. It won't affect behavior, however our logs will not be able to aggregate such requests by client library version.